code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm

```diff
@@ -227,3 +227,14 @@
 	)
 	result = /obj/item/food/meatclown
 	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/lasagna
+	name = "Lasagna"
+	reqs = list(
+		/obj/item/food/meat/cutlet = 2,
+		/obj/item/reagent_containers/food/snacks/grown/tomato = 1,
+		/obj/item/reagent_containers/food/snacks/cheesewedge = 2,
+		/obj/item/food/spaghetti
```

Comment:
This right here is why you shouldn't do "web edits", AKA using github.com to make commits.

It's a way of showing to us that you didn't test your code. In this case, you web edited it to use spaghetti instead of whatever it was before, but forgot to put the `= 1` necessary to make it actually work.

```suggestion
		/obj/item/food/spaghetti = 1
```

Reply:
Thanks for pointing that out. I figured it would be fine, but obviously not lol. Will make sure to double check in the future. Also will change the changelog to be for players.